editor/src/components/canvas/canvas-strategies/strategies/absolute-duplicate-strategy.spec.browser2.tsx

```diff
@@ -105,4 +114,100 @@ describe('Absolute Duplicate Strategy', () => {
       `),
     )
   })
+  describe('with fragments', () => {
+    setFeatureForBrowserTests('Fragment support', true)
+    it('duplicates the selected absolute element when pressing alt, even if it is a fragment', async () => {
+      const renderResult = await renderTestEditorWithCode(
+        formatTestProjectCode(
+          projectWithFragment(`<React.Fragment data-uid='fragment'>
+        <div
+          style={{
+            backgroundColor: '#d089cc',
+            width: 150,
+            height: 186,
+            contain: 'layout',
+            left: 7,
+            top: 186,
+            position: 'absolute',
+          }}
+          data-uid='child'
```

Comment:
Very minor, but can you change the uid to 3 characters please in this test, since the duplicate will have a 3 character uid

Reply:
done!